Amplify/DevMenu/View/DetailViewFactory.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// A factory to create detail views based on `DevMenutItem`
+@available(iOS 13.0, *)
+public class DetailViewFactory {
+
+    static func getDetailView(devMenuItemLabel: DevMenuList.ItemLabel) -> AnyView {
```

Comment:
Getting a new view based on label doesnot looks right. Maybe move the enum DevMenuList.ItemLabel out from DevMenuList and call it DevMenuItemType ?